lib/utils/emailService/mailler.js

```
@@ -0,0 +1,34 @@
+const sgMail = require('@sendgrid/mail'),
+  constants = require('../../../constants');
+
+// Set sendgrid API key from env
+sgMail.setApiKey(process.env.SENDGRID_API_KEY);
+
+/**
+ * sends E-mail.
+ * @param {string} to Receiver of mail.
+ * @param {string} subject Subject of Mail.
+ * @param {html} html Html file
+ * @returns {promise} Promise
+ */
+const sendMail = (to, subject, html) => {
+  const msg = {
+    to,
+    from: constants.from,
+    subject,
+    html,
+  };
+
+  if (!to && !subject && !html) {
+    return 'Parameters are expected';
```

It is better to throw an error here than returning a string describing the error.

---

Noted

---

But when throwing an error isn't a message require?